3498/3/app/helpers/places_helper.rb

```diff
@@ -0,0 +1,27 @@
+module PlacesHelper
+  def load_place_info
+    session[:place_name] = params[:place_name].to_s
+    @place = Place.find_by(name: params[:place_name].to_s)
+    session[:place_id] = @place.id
+  end
+
+  def rate_place
+    redirect '/login' unless login?
+    add_comment
+  end
+
+  def add_comment
+    @comment = Comment.new(rating: params[:rate].to_i,
```

Comment:
bad idea :) try to scope to user as it was before, just replace create with new:
https://stackoverflow.com/questions/1253426/what-is-the-difference-between-build-and-new-on-rails/1253462